applications/GeoMechanicsApplication/custom_workflows/dgeosettlement.cpp

```diff
@@ -69,6 +69,11 @@ double GetIncreaseFactorFrom(const Parameters& rProjectParameters)
     return rProjectParameters["solver_settings"]["increase_factor"].GetDouble();
 }
 
+double GetMaxDeltaTimeFactorFrom(const Parameters& rProjectParameters)
+{
+    return rProjectParameters["solver_settings"]["max_delta_time_factor"].GetDouble();
```

Comment:
Are you sure this is the correct location in the project parameters `.json` file? When I look at lines 116-117 of file `geomechanics_analysis.py`, it appears to me that we are missing an intermediate field: `["solver_settings"]["time_stepping"]["max_delta_time_factor"]`. If that is correct, then we may also want to add a small test for that, since it was not detected by the test suite.

Reply:
Very good find, thank you! This has to do with the inconsistent setup of the .json file. Within ["time_stepping"] only 2 items are defined of which this is one. All the others ( like the decrease factor, increase factor, min number of iterations ) are one level higher. To get a more logical structure, I would like to reorder this at some point.  For now, I have checked by enabeling the settlement workflow test during developing, without this fix it fails.